app/tests/unit/components/Form/ProjectMilestoneReportFormSummary.test.tsx

```diff
@@ -0,0 +1,166 @@
+                  projectId: 1,
+                  reportDueDate: "2020-01-03T23:59:59.999-07:00",
+                  reportingRequirementIndex: 1,
+                },
+                operation: "UPDATE",
+                formChangeByPreviousFormChangeId: {
+                  newFormData: {
+                    comments: "Not updated comment",
+                    projectId: 1,
+                    reportDueDate: "2020-01-03T23:59:59.999-07:00",
+                    reportingRequirementIndex: 1,
+                  },
+                },
+              },
+            },
+            {
+              node: {
+                id: "Test ID - 3",
+                isPristine: false,
+                newFormData: {
+                  comments: "Added comment",
+                  projectId: 1,
+                  reportDueDate: "2020-01-04T23:59:59.999-07:00",
+                  reportingRequirementIndex: 1,
+                },
+                operation: "CREATE",
+                formChangeByPreviousFormChangeId: null,
+              },
+            },
+            {
+              node: {
+                id: "Test ID - 4",
+                isPristine: false,
+                newFormData: {
+                  comments: "Removed comment",
+                  projectId: 1,
+                  reportDueDate: "2020-01-05T23:59:59.999-07:00",
+                  reportingRequirementIndex: 1,
+                },
+                operation: "ARCHIVE",
+                formChangeByPreviousFormChangeId: {
+                  newFormData: {
+                    comments: "Removed comment",
+                    projectId: 1,
+                    reportDueDate: "2020-01-05T23:59:59.999-07:00",
+                    reportingRequirementIndex: 1,
+                  },
+                },
+              },
+            },
+          ],
+        },
+      };
+    return result;
+  },
+};
+
+const defaultComponentProps = {
+  setValidatingForm: jest.fn(),
+  onSubmit: jest.fn(),
+};
+
+const componentTestingHelper =
+  new ComponentTestingHelper<ProjectMilestoneReportFormSummaryQuery>({
+    component: ProjectMilestoneReportFormSummary,
+    testQuery: testQuery,
+    compiledQuery: compiledProjectMilestoneReportFormSummaryQuery,
+    getPropsFromTestQuery: (data) => ({
+      query: data.query,
+      projectRevision: data.query.projectRevision,
+    }),
+    defaultQueryResolver: mockQueryPayload,
+    defaultQueryVariables: {},
+    defaultComponentProps: defaultComponentProps,
+  });
+
+describe("The Project Milestone Report Form Summary", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    componentTestingHelper.reinit();
+  });
+
+  it("Only displays the data fields that have changed", () => {
+    componentTestingHelper.loadQuery();
+    componentTestingHelper.renderComponent();
+
+    // First Node
+    expect(screen.getByText("Updated Test comment")).toBeInTheDocument();
+    expect(screen.getByText(/Jan[.]? 2, 2020/)).toBeInTheDocument();
```

Comment:
These assertions are the same as in the test below--maybe in this test it would be better to look for the field labels you'd expect to see/not see? `ProjectManagerFormSummary.test.tsx` does it that way